index.html

```diff
@@ -0,0 +1,551 @@
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>DISTANT COFFEE</strong>  At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled.
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>COFFEE EVERYWHERE</strong>  Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanation.
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <p>
+                                <strong>STILL NOT ENOUGH TIME</strong>  There has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>INFLATION</strong>  solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>FLUCTUATION</strong>  Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>EXPANSION</strong>  Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.
+                            </p>
+                        </td>
+                    </tr>
+                </table>
```

Comment:
All elements should be aligned using float.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/float_requirement.png)